libs/core/src/lib/nested-list/nested-list/nested-list.directive.ts

```diff
@@ -123,4 +153,26 @@ export class NestedListDirective implements AfterContentInit, NestedListInterfac
         const filteredParentElements = parentElements.filter((_element) => _element.hasAttribute('fd-nested-list'));
         return filteredParentElements.length;
     }
+
+    /** @hidden */
+    private _setAriaLevel(level: number): void {
+        this.nestedItems.forEach((item) => (item._ariaLevel = level));
```

Comment:
I didn't get this.
Here we are setting aria-level for fd-nested-list-item.
As far as I know arial-level has to be used along with role="heading" to reflect h1 h6 tags meaning